lib/codegen/src/print_errors.rs

```diff
@@ -28,11 +28,11 @@ pub fn pretty_verifier_error<'a>(
 
     while i != errors.len() {
         if let ir::entities::AnyEntity::Inst(_) = errors[i].location {
+            i += 1;
+        } else {
             let err = errors.remove(i);
 
-            writeln!(w, "Miscellaneous error: {}\n", err).unwrap()
-        } else {
-            i += 1;
+            writeln!(w, "misc verifier {}\n", err).unwrap()
```

Comment:
Can you comment on this wording change? "Miscellaneous error: {}" isn't the most descriptive, but "misc verifier {}" without a colon seems like it could be confusing.

Reply:
Before this change, "verifier error summary: {}" was shown at the beginning of the function body when an error was not related to an instruction, but now I think the wording should be changed, since more than one error can be shown this way. Furthermore, with "Miscellaneous error: {}", there are two colons in a row separated by a word, since error messages start with "{entity}:" when displayed.

So in the end I actually changed it in order to look more like what we have for instructions, where the format is "^~~~ verifier {}". 

If you have any other suggestion though, I'll gladly take it!